pkg/helm/helm.go

```diff
@@ -98,8 +98,12 @@ func (r *Render) GenerateHelmValues(input interface{}) (map[string]interface{},
 	typeOf := reflect.TypeOf(input)
 	for i := 0; i < valueOf.NumField(); i++ {
 		helmValueKey := typeOf.Field(i).Tag.Get(helmFieldTag)
-		if helmValueKey != "" && valueOf.Field(i).Len() > 0 {
-			rawArgs = append(rawArgs, fmt.Sprintf("%s=%s", helmValueKey, valueOf.Field(i)))
+		if len(helmValueKey) > 0 && valueOf.Field(i).Len() > 0 {
+			if helmValueKey == "*" {
```

Comment:
todo: I think it's necessary to write the unit test for `*` case.

Reply:
got it

Reply:
done